source/val/validation_state.h

```diff
@@ -255,6 +260,9 @@ class ValidationState_t {
   /// IDs that are entry points, ie, arguments to OpEntryPoint.
   std::vector<uint32_t> entry_points_;
 
+  /// Functions IDs that are target of OpFunctionCall.
+  std::vector<uint32_t> function_call_targets_;
```

Comment:
It might also be true that entry_points could be an unordered map.  But that would be for another MR.